### ISSUE TITLE
Update Skadi tiler to work alongside render_pyramid

### ISSUE THREAD

**Issue by iandees** [mojodna/marblecutter#49]
I haven't confirmed that the [`render_bulk_skadi.py` script](https://github.com/mojodna/marblecutter/blob/master/examples/render_bulk_skadi.py) works, and we need to make sure it will run properly in Batch mode alongside the mercator pyramids.

**Comment by iandees** [comment]
I've been working on this over the last day or two. The existing bulk script didn't support multiple machines, so I'm modifying it to support Batch and spreading work across multiple machines.

**Comment by iandees** [comment]
I ended up doing the work on a branch from before you split the repo apart, @mojodna. Branch is here: https://github.com/mojodna/marblecutter/compare/tilezen

**Comment by iandees** [comment]
I ran Skadi tiles last night for the world based on the code in the branch above and it worked well. In my initial testing I was using 4GB of RAM and it was failing due to memory errors. I bumped it up to 7GB and they all completed successfully.

There was one job that seemed to get stuck and was running when I checked this morning. I canceled and re-ran that job and it completed in a couple minutes this morning.

This global run was done into `s3://elevation-tiles-dev/20170829/...`, so when the global run happens I'll have to point it to the prod tiles bucket.
